Plugins/BCClient/Source/BCClientPlugin/Private/BrainCloudWrapper.cpp

```diff
@@ -216,6 +216,13 @@ void UBrainCloudWrapper::authenticateAdvanced(EBCAuthType in_authenticationType,
     _client->getAuthenticationService()->authenticateAdvanced(in_authenticationType, in_ids, in_forceCreate, in_extraJson, in_callback);
 }
 
+void UBrainCloudWrapper::authenticateUltra(const FString& in_ultraUsername, const FString& in_ultraIdToken, bool in_forceCreate, IServerCallback* in_callback)
+{
+    _authenticateCallback = in_callback;
+    initializeIdentity();
+    _client->getAuthenticationService()->authenticateUltra(in_ultraUsername, in_ultraIdToken, in_forceCreate, in_callback);
```

Comment:
last parameter should be `this` I believe here. Called from the wrapper. That's why they are caching `_authenticateCallback = in_callback`